kotlin/src/main/kotlin/com/example/android/learn-kotlin/java/com/example/kotlin_samples/KotlinSamples.kt

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.kotlin_samples
```

Comment:
I think underscore characters are not recommended in package names, so let's rename. Maybe something like
com.example.android.basics (to be similar to the package name for the other kotlin snippets in com.example.android.coroutines.testing) and make sure the folder structure is updated too